mariadb/content.md

```diff
@@ -63,7 +74,7 @@ $ docker run --detach --network some-network --name some-application --env APP_D
 The following command starts another `%%IMAGE%%` container instance and runs the `mariadb` command line client against your original `%%IMAGE%%` container, allowing you to execute SQL statements against your database instance:
 
 ```console
-$ docker run -it --network some-network --rm %%IMAGE%% mariadb -hsome-%%REPO%% -uexample-user -p
+$ docker run -it --network some-network --rm %%IMAGE%% mariadb -h some-%%REPO%% -u example-user -p example
```

Comment:
for weird historical reasons (I choose not to look at) there isn't a space between `-p` and the password "example". As you worked out, without a password it just prompts for the password. Isn't it safer to not have a password on a command? If its scripting chances are you don't need the `-it` (though would need for `mariadb` a `-e` or something to execute.

So some additional clarity why this change perhaps with a note on constraints comments.

Reply:
@grooverdan I removed the -p and left the user with just the name example-user.